src/main/java/seedu/address/model/commons/NameContainsKeywordsPredicate.java

```diff
@@ -6,28 +6,31 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.model.travelplan.TravelPlan;
 
+
 /**
  * Tests that a {@code TravelPlanObject}'s or {@code TravelPlan}'s {@code Name} matches any of the keywords given.
  */
-public class NameContainsKeywordsPredicate implements Predicate<Object> {
+public class NameContainsKeywordsPredicate implements Predicate<TravelPlan> {
```

Comment:
Why this one change back to TravelPlan? It's not used in other place meh? like Activity all that